src/__experimental__/components/textarea/textarea.component.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import { InputPresentation } from '../input';
+import FormField from '../form-field';
+import CharacterCount from './character-count';
+import TextareaInput from './textarea-input.component';
+
+const i18nNumberOpts = { precision: 0 };
+
+class Textarea extends React.Component {
+  static propTypes = {
```

Comment:
Can we move the static propTypes and defaultProps to bottom of the component?
```js
Textarea.propTypes = {}
Textarea.defaultProps = {}
```

Reply:
Sure, good point. Thanks